.changeset/large-words-refuse.md

```
@@ -0,0 +1,11 @@
+---
+'@mastra/playground-ui': patch
+---
+
+Search the spans of a trace from the timeline in Studio
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)